spyder/config/manager.py

```diff
@@ -25,18 +25,13 @@
 from spyder.config.main import CONF_VERSION, DEFAULTS, NAME_MAP
 from spyder.config.types import ConfigurationKey, ConfigurationObserver
 from spyder.config.user import UserConfig, MultiUserConfig, NoDefault, cp
+from spyder.plugins.shortcuts.utils import SHORTCUTS_FOR_WIDGETS_DATA
 from spyder.utils.programs import check_version
 
 
 logger = logging.getLogger(__name__)
 
-EXTRA_VALID_SHORTCUT_CONTEXTS = [
-    '_',
-    'array_builder',
-    'console',
-    'find_replace',
-    'switcher'
-]
+EXTRA_VALID_SHORTCUT_CONTEXTS = ['_', 'find_replace']
```

Comment:
I guess those other contexts were previously removed and are not needed anymore, right?

Reply:
Correct. `array_builder` is no longer used, `console` was replaced `internal_console` (the actual plugin name) and `switcher` is now a plugin, so we don't need it as an extra context.